features/aborting.yml

```diff
@@ -2,3 +2,12 @@ name: AbortController and AbortSignal
 description: The `AbortController` and `AbortSignal` APIs allow you to cancel an ongoing operation, such as a `fetch()` request.
 spec: https://dom.spec.whatwg.org/#aborting-ongoing-activities
 caniuse: abortcontroller
+compat_features:
+  - api.AbortSignal
+  - api.AbortSignal.abort_event
+  - api.AbortSignal.abort_static
+  - api.AbortSignal.abort_static.reason_parameter
+  - api.AbortSignal.aborted
+  - api.AbortSignal.reason
+  - api.AbortSignal.throwIfAborted
+  - api.AbortSignal.timeout_static
```

Comment:
This is (still?) wiping out a bunch of keys from BCD and explicitly mentioned in the description. What's the story here?

Reply:
Ugh, I thought I'd added the difference back in but missed the AbortController section. Added back.